code/modules/food_and_drinks/recipes/tablecraft/recipes_table.dm

```diff
@@ -252,4 +252,18 @@
 	)
 	tools = list(/obj/item/kitchen/sushimat)
 	result = /obj/item/reagent_containers/food/snacks/sliceable/Tai_maki
+	category = CAT_FOOD
+
+/datum/crafting_recipe/food/cak
```

Comment:
To get the brain transferring to work, you need to add this, above the cak's crafting recipe:

```
/datum/crafting_recipe/food

/datum/crafting_recipe/food/New()
	parts |= reqs
```